go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	k8s.io/apimachinery v0.21.0
 	k8s.io/cli-runtime v0.21.0
 	k8s.io/client-go v0.21.0
-	k8s.io/helm v2.16.12+incompatible
+	k8s.io/helm/v3 v3.6.1
```

Comment:
This will never work as we still need the k8s.io/helm pkg for helm v2 support and `k8s.io/helm/v3` is for helm 3.

Reply:
How long do you plan to support helm v2?